### ISSUE TITLE
available memory dropped to 258.3mb after kernal flash 

### ISSUE THREAD

**Issue by ckingmuzic** [ClusterM/hakchi2#577]
after flashing the custom kernel the memory dropped down to 258mb from 300mb is that right?

Snes Mini - usa

**Comment by ajmi982** [comment]
Probably because of the mirroring of the original 21 games in the Hakchi folder. 

Sent from my iPhone. Please pardon any spelling or grammar mistakes. 

On Oct 8, 2017, at 4:39 PM, ckingmuzic <notifications@github.com> wrote:

after flashing the custom kernel the memory dropped down to 258mb from 300mb is that right?

**Comment by DecanFrost** [comment]
I thought the same thing but if you Deselect the built in games you will see it shows less memeory used but cap is still 258mb

**Comment by Kumoashi** [comment]
I don't believe that Hakchi2 can read the amount of free space left on your SNES Classic until you flash the custom kernel. 300MB is just an estimate of how much space is available.
It's like when you buy a new hard drive. It might say something like 500GB, but that is not how much you get. It ends up being more like 462GB.

**Comment by JohnSilver78** [comment]
It happened to me exactly the same, but I uninstalled everything (using the hackhi tool) and arranged matter. In fact the application marked memory problems, and even though I deleted games, I could not get the excess memory to disappear until I uninstalled.

**Comment by ckingmuzic** [comment]
JohnSliver78 did that resolve it to give you more room?

**Comment by JohnSilver78** [comment]
Yes that resolve everything to me. :) Now i can play without memory problems.

**Comment by Andreunto** [comment]
I have the same problem here. I tried uninstalling hakchi on the snes and format the memory but the space is always 258mb.

How can I get back to 300mb?

**Comment by K4Z1** [comment]
Mine's down to 201,5MB... oh well :\

**Comment by false-9** [comment]
196.6MB here, whys this happening?

**Comment by ajmi982** [comment]
I think the memory now reads (used) / (remaining) and if u add those two numbers that is your Total 

So 

50mb / 190mb is 240mb of usable space and u have already used 50

Or that 190mb number now accounts of save states to avoid the C8 errors. 

Idk. Speculating. 

Sent from my iPhone. Please pardon any spelling or grammar mistakes. 

On Oct 11, 2017, at 10:15 AM, false-9 <notifications@github.com> wrote:

196.6MB here, whys this happening?

**Comment by K4Z1** [comment]
I've got 95 MB / 162 MB now in hakchi2.21b.
Adding & removing original games does nothing (not much), as promised.

I deleted all (~51 MB) savegames: I got 95 MB / 213 MB.

I guess there never was 300 MB, but 258 MB in reality. Ok.
So without savegames I should have 258 MB? But I got only 213 MB. So there's 45 MB missing... where are they? Are they reserved for the original games? It's bad for people who don't use them.

Btw, I just saw in FTP: "bgm_home.wav" and "bgm_boot.wav" ...why is it the same? wav files are huge, maybe we could save some MB there?
